docs/management/admin/endpoint-artifacts.asciidoc

```diff
@@ -1,4 +1,38 @@
 [[endpoint-artifacts]]
+[chapter]
 = Optimize {elastic-defend}
 
-This page is a placeholder for future documentation.
+If you encounter problems like incompatibilities with other antivirus software, too many false positive alerts, or excessive storage or CPU usage, you can optimize {elastic-defend} to mitigate these issues.
+
+Endpoint artifacts — such as trusted applications and event filters — and Endpoint exceptions let you modify the behavior and performance of _{elastic-endpoint}_, the component installed on each host that performs {elastic-defend}'s threat monitoring, prevention, and response actions.
+
+The following table explains the differences between several Endpoint artifacts and exceptions, and how to use them for specific purposes:
+
+[cols="2"]
+|===
+
+| <<trusted-apps-ov,Trusted applications>>
+a| *_Prevents {elastic-endpoint} from monitoring a process._* Use to avoid conflicts with other software, usually other antivirus or endpoint security applications.
+
+* Creates intentional blind spots in your security environment — use sparingly!
+* Does not monitor the application for threats, and does not create alerts for the application, even if it behaves like malware, ransomware, etc.
+* Does not generate events for the application except process events for visualizations.
+* May improve performance, since {elastic-endpoint} monitors fewer processes.
+* May still generate malicious behavior alerts, if the application's process events indicate malicious behavior.
```

Comment:
Yes, the minimal process events that still get created could possibly trigger the [malicious behavior protection](https://www.elastic.co/guide/en/security/master/configure-endpoint-integration-policy.html#behavior-protection) feature, probably because it's looking more at behavior patterns in the data, regardless of the application/process doing the behavior. Malicious behavior protection is separate from other Elastic Defend features that presumably would NOT trigger for a trusted application (such as malware, ransomware, or memory threat protections, etc.), since they wouldn't be looking at process events in the same way.

The "for visualizations" detail might be unnecessary and distracting here; we could probably just say "for internal use" or similar.

@caitlinbetz @kevinlog @ferullo Any additional details on what might trigger these malicious behavior alerts? I get the sense that we're mentioning this because it's happened to customers when they weren't expecting alerts, but any other info that would help?

Reply:
@kevinlog Thanks for the extra context! I'll add a cross-reference to Endpoint alert exceptions as you suggest.

@nastasha-solomon I think this will resolve the point of confusion that customers may be having; it sounds like they might not need the additional explanation of how the alerts are still being created, just how they can stop them (create an exception).